bindata/network/ovn-kubernetes/common/002-rbac.yaml

```diff
@@ -133,3 +133,178 @@ subjects:
+metadata:
+  name: openshift-ovn-kubernetes-node-extra
+rules:
+- apiGroups: [""]
+  resources:
+  - namespaces
+  - nodes
+  - pods
+  verbs:
+  - get
+  - list
+  - patch
+  - watch
+  - update
+- apiGroups: [""]
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+  - patch
+  - watch
+  - delete
+- apiGroups: [""]
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - create
+  - update
+  - patch
+- apiGroups: [""]
+  resources:
+  - services
+  - endpoints
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - discovery.k8s.io
+  resources:
+  - endpointslices
+  verbs:
+  - list
+  - watch
+- apiGroups: ["networking.k8s.io"]
+  resources:
+  - networkpolicies
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: ["", "events.k8s.io"]
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+  - update
+- apiGroups: ["security.openshift.io"]
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
+  resourceNames:
+  - privileged
+- apiGroups: [""]
+  resources:
+  - "nodes/status"
+  - services
+  verbs:
+  - patch
+  - update
+- apiGroups: ["k8s.ovn.org"]
+  resources:
+  - egressfirewalls
+  - egressips
+  - egressqoses
+  - adminpolicybasedexternalroutes
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - patch
+- apiGroups: ["cloud.network.openshift.io"]
+  resources:
+  - cloudprivateipconfigs
+  verbs:
+  - create
+  - patch
+  - update
+  - delete
+  - get
+  - list
+  - watch
+- apiGroups: ["apiextensions.k8s.io"]
+  resources:
+  - customresourcedefinitions
+  verbs:
+    - get
+    - list
+    - watch
+- apiGroups: ['authentication.k8s.io']
+  resources: ['tokenreviews']
+  verbs: ['create']
+- apiGroups: ['authorization.k8s.io']
+  resources: ['subjectaccessreviews']
+  verbs: ['create']
+- apiGroups:
+  - k8s.cni.cncf.io
+  resources:
+  - network-attachment-definitions
+  - multi-networkpolicies
+  verbs: ["list", "get", "watch"]
+
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: openshift-ovn-kubernetes-node-extra
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: openshift-ovn-kubernetes-node
+subjects:
+- kind: ServiceAccount
+  name: ovn-kubernetes-node
+  namespace: openshift-ovn-kubernetes
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: openshift-ovn-kubernetes-sbdb
+  namespace: openshift-ovn-kubernetes
+rules:
+- apiGroups: [""]
+  resources:
+  - endpoints
+  verbs:
+  - create
+  - update
+  - patch
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - create
+  - get
+  - list
+  - update
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: openshift-ovn-kubernetes-sbdb
+  namespace: openshift-ovn-kubernetes
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: openshift-ovn-kubernetes-sbdb
+subjects:
+- kind: ServiceAccount
+  name: ovn-kubernetes-node
+  namespace: openshift-ovn-kubernetes
+  
```

Comment:
I think this needs to be changed. 

I believe that it needs to look like this 
```
subjects:
- kind: ServiceAccount
  name: ovn-kubernetes-controller
  namespace: openshift-ovn-kubernetes
```
everything that uses the `leases` resource is using the `ovn-kubernetes-controller` ServiceAccount

Reply:
@trozet this will fix the leader election issues you pointed out in the CI.

Reply:
@JacobTanenbaum @ricky-rav this seems a bit confusing to me. If I look at [003-rbac-controller.yaml](https://github.com/ricky-rav/cluster-network-operator/blob/8c78ea5e2f1c8ef329e721ba2309b47ce609b12c/bindata/network/ovn-kubernetes/common/003-rbac-controller.yaml#L146-L182) isn't that all needed? If `openshift-ovn-kubernetes-controller` means cluster manager, why would we need access to the dbs anyways?

Reply:
Ref issue in CI JOB:

- E0713 23:06:08.418654 69905 leaderelection.go:327] error retrieving resource lock openshift-ovn-kubernetes/ovn-kubernetes-master-ip-10-0-136-147.ec2.internal: [leases.coordination.k8s.io](http://leases.coordination.k8s.io/) "ovn-kubernetes-master-ip-10-0-136-147.ec2.internal" is forbidden: User "system:serviceaccount:openshift-ovn-kubernetes:ovn-kubernetes-controller" cannot get resource "leases" in API group "[coordination.k8s.io](http://coordination.k8s.io/)" in the namespace "openshift-ovn-kubernetes"
	- Happening in all jobs (PR1874)
	- https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-network-operator/1874/pull-ci-openshift-cluster-network-operator-master-e2e-aws-ovn-windows/1679603533172707328/artifacts/e2e-aws-ovn-windows/gather-extra/artifacts/pods/openshift-ovn-kubernetes_ovnkube-node-bv6vr_ovnkube-controller_previous.log 
	- https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-network-operator/1874/pull-ci-openshift-cluster-network-operator-master-e2e-aws-ovn-windows/1679603533172707328